docs/migrating_to_20_1.md

```diff
@@ -1,3 +1,106 @@
 # Migrating to Bento v20.1
 
-TODO
+## Minio to Garage Migration
+
+### Prerequisites
+Before starting the migration, ensure you have:
+- Access to both Minio and Garage instances
+- `s3cmd` installed and configured
+- Sufficient local disk space for the backup
+
+### Step 1: Back up data from Minio
+
+#### Option A: If you have `~/.s3cfg-minio-local`
+```bash
+# Sync data from Minio to local backup
+s3cmd -c ~/.s3cfg-minio-local sync s3://drop-box/ ./drop-box-backup/
+s3cmd -c ~/.s3cfg-minio-local sync s3://drs/ ./drs-backup/
+```
+
+#### Option B: Without existing config file
+```bash
+# Backup drop-box bucket
+s3cmd --access_key=AKIA_MINIO --secret_key=MINIOSECRET \
+      --host=minio.bentov2.local --no-check-certificate \
+      --host-bucket="minio.bentov2.local/%(bucket)s" \
+      sync s3://drop-box/ ./drop-box-backup/
+
+# Backup drs bucket
+s3cmd --access_key=AKIA_MINIO --secret_key=MINIOSECRET \
+      --host=minio.bentov2.local --no-check-certificate \
+      --host-bucket="minio.bentov2.local/%(bucket)s" \
+      sync s3://drs/ ./drs-backup/
+```
+
+### Step 2: Setting up garage
```

Comment:
garage -> Garage

also, there needs to be instructions here that NOW and only now can we check out the v20.1 tag, as if we do so any earlier we lose minio access